components/gallery/GalleryItemMoreActionBtn.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <b-dropdown aria-role="list" position="is-bottom-left">
+    <template #trigger>
+      <NeoButton label="⋮" />
+    </template>
+
+    <b-dropdown-item aria-role="listitem" @click="downloadMedia"
```

Comment:
I just saw that it doesn't work with 3D files
I think there is always an image on Token, any idea how you would like to handle this case?
we can still do something like `v-if="currentGalleryItemImage.value.image"` but I'm not sure if it will do the work

Reply:
Nevermind I will use the mimeType which includes `image/*`

Reply:
@preschian any idea why `$store.getters['history/getCurrentlyViewedItem']` is not updating in component? 
While when I open vue Devtools, I can see state updated